model/src/main/java/org/cbioportal/model/GenesetAlteration.java

```diff
@@ -0,0 +1,40 @@
+package org.cbioportal.model;
+
+import java.io.Serializable;
+
+public class GenesetAlteration implements Serializable {
```

Comment:
Actually yeah you can leave `geneticEntityId` out for now as we don't need it, just `values` would be enough in the parent class.